client/me/help/help-contact/index.jsx

```diff
@@ -247,6 +255,10 @@ module.exports = React.createClass( {
 
 		if ( olark.details.isConversing && olark.isOperatorAvailable ) {
 			return <OlarkChatbox />;
+		} else {
+			analytics.tracks.recordEvent( 'calypso_help_offline_form_display', {
+				form_type: showKayakoVariation ? 'kayako' : 'forum'
+			} );
```

Comment:
@omarjackman thanks for that suggestion. I've updated, and it works better as implemented now. One thing I'm not sure about is defining all the `const`s here. It feels a bit like I'm repeating code from elsewhere, but I don't know if there's a better way. Do you think it's reasonable or is there a better approach I might try?